drivers/ethernet/Kconfig.nxp_enet

```diff
@@ -0,0 +1,52 @@
+# NXP ENET Ethernet driver configuration options
+
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ETH_NXP_ENET
+	bool "NXP ENET Ethernet driver"
+	default y
+	depends on DT_HAS_NXP_ENET_MAC_ENABLED
+	select NOCACHE_MEMORY if HAS_MCUX_CACHE
+	select ARM_MPU if CPU_CORTEX_M7
+	select MDIO if DT_HAS_NXP_ENET_MDIO_ENABLED
+	select EXPERIMENTAL
+	help
+	  Enable NXP ENET Ethernet driver.
+
+if ETH_NXP_ENET
+
+config ETH_NXP_ENET_USE_DTCM_FOR_DMA_BUFFER
+	bool "Use DTCM for hardware DMA buffers"
+	default y
+	help
+	  Place the hardware DMA buffers into DTCM for better
+	  networking performance.
+
+config ETH_NXP_ENET_HW_ACCELERATION
+	bool "Hardware acceleration for ethernet checksums"
```

Comment:
Seems like this should be enabled by default?

Reply:
no, it doesn't work with ipv6